bundles/org.openhab.binding.homematic/README.md

```diff
@@ -177,8 +180,7 @@ If set to true, devices are automatically unpaired from the gateway when their c
 **Warning!** The option "factoryResetOnDeletion" also unpairs a device, so in order to avoid unpairing on deletion completely, both options need to be set to false! (default = false)
 
 -   **factoryResetOnDeletion**
-If set to true, devices are automatically factory reset when their corresponding things are removed.
-Due to the factory reset, the device will also be unpaired from the gateway, even if "unpairOnDeletion" is set to false! (default = false)
+If set to true, devices are automatically factory reset when their corresponding things are removed. Due to the factory reset, the device will also be unpaired from the gateway, even if "unpairOnDeletion" is set to false! (default = false)
```

Comment:
Why did you join those sentences, AFAIK its prefered to have once line per sentence?

Reply:
@martinvw I was considering it, but I gave the rule checker a higher prio than the "one line per sentence" guideline.
If you like, I can revert this part of the change again.

Reply:
Yes please undo. @Confectrician is that a false positive of the checks which should be fixed?

Reply:
I added the \n to line 181 again